netcore/Makefile.am

```diff
@@ -89,20 +89,25 @@ check: $(addprefix run-, $(TEST_SUITES))
 clean:
 	rm -rf sdk shared host dotnet tests LICENSE.txt ThirdPartyNotices.txt $(NETCORESDK_FILE)
 
+COREFX_ROOT=/prj/corefx-egor
```

Comment:
While I appreciate default values I think you should avoid this one ;-)

Reply:
@filipnavara oops :)